thavalon-webapp/src/App.tsx

```diff
@@ -73,8 +73,17 @@ function App() {
         <Route path="/register" render={() => registerPage()} />
         <Route path="/game" />
       </Switch>
-      <Login setLoggedIn={setLoggedIn} setShowLoginModal={setShowLoginModal} showLoginModal={showLoginModal} />
-      <CreateJoinGameModal show={showCreateJoinGameModal} setOpen={setShowCreateJoinGameModal} />
+      {showLoginModal &&
+        <Login
+          setLoggedIn={setLoggedIn}
+          setShowLoginModal={setShowLoginModal}
+          showLoginModal={showLoginModal} />}
+      {showCreateJoinGameModal &&
+        <CreateJoinGameModal
+          isLoggedIn={loggedIn}
+          setShowLoginModal={setShowLoginModal}
+          show={showCreateJoinGameModal}
+          setOpen={setShowCreateJoinGameModal} />}
```

Comment:
Should we still pass in show here? Not sure if needed, but at least both login and createJoinGameModal should both set or not set show

Reply:
nvm we just call it two separate things

Reply:
Login does have this as well. Not sure if it's useful anywhere, but I think it is.